src/css/components/SelectCard.scss

```diff
@@ -0,0 +1,30 @@
+.Card {
+    border-radius: 5px;
+    width: 15%;
```

Comment:
We may want to set this to `width: 100%` and `height: auto` for maximum flexibility. Then use a wrapping element like `Grid` or just [`u-flex` utilities](https://teamsnap-ui-patterns.netlify.app/patterns/utilities/flex.html) to quickly compose elements on the page in different layouts.